docs/contributing.md

```diff
@@ -3,48 +3,93 @@
 Thank you for being interested in contributing to the `sphinx-book-theme`! You
 are awesome ✨.
 
-This page contains developer documentation to help you get started.
-
 ```{toctree}
 :hidden:
 contributing-ebp
 ```
 
-## The EBP Contributing Guide
+This project follows the `executablebooks/` contributors guide, which you can
+find [here](contributing-ebp). It contains information about our conventions
+around coding style, pull request workflow, commit messages and more.
```

Comment:
[`e8fd282` (#172)](https://github.com/executablebooks/sphinx-book-theme/pull/172/commits/e8fd2821ab862613e0c4863a0f194ed4fd1f3562)

Should we drop the section that this commit is changing then? It was added in #163.

Reply:
oh right I didn't see that. Well I would say yes, because I don't see a need to have to specifically include it in this documentation (which among other things means it won't always be up to date with the "original", if that gets altered). But I guess @choldgraf added it so should get he's opinion first...

Reply:
I'd add this suggestion now though, and we can do that change in another PR

Reply:
Or... I could just add another commit doing this. :)